Corona-Warn-App/src/main/res/layout/fragment_scan_qr_code.xml

```diff
@@ -32,19 +32,29 @@
         app:zxing_viewfinder_laser_visibility="false" />
 
     <TextView
-        android:id="@+id/check_in_qr_code_scan_body"
+        android:id="@+id/qr_code_scan_body"
         style="@style/qrCodeScanBody"
         android:layout_width="@dimen/scan_qr_code_viewfinder_size"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/scan_qr_code_viewfinder_center_offset"
         android:text="@string/qr_code_scan_body"
-        app:layout_constraintEnd_toEndOf="@id/check_in_qr_code_scan_preview"
-        app:layout_constraintStart_toStartOf="@id/check_in_qr_code_scan_preview"
-        app:layout_constraintTop_toBottomOf="@id/check_in_qr_code_scan_guideline_center" />
+        app:layout_constraintEnd_toEndOf="@id/qr_code_scan_preview"
+        app:layout_constraintStart_toStartOf="@id/qr_code_scan_preview"
+        app:layout_constraintTop_toBottomOf="@id/qr_code_scan_guideline_center" />
 
+    <ProgressBar
```

Comment:
Can we use `CircularProgressIndicator ` from MDC ? it has a nice hide and show animations?